utils.go

```diff
@@ -41,6 +41,27 @@ func CreateResourceLabels(resourceID string) map[string]string {
 	return labels
 }
 
+func CreateAllResourceLabelsFrom(rm resourceMeta) map[string]string {
+	labels := make(map[string]string)
+	split := strings.Split(rm.ResourceURL, "/")
+	labels["resource_group"] = split[4]
```

Comment:
would you able to put constants for this numbers, and add some comment ?

Reply:
done